checks/raw/binary_artifact.go

```diff
@@ -40,6 +44,27 @@ func BinaryArtifacts(c clients.RepoClient) (checker.BinaryArtifactData, error) {
 		return checker.BinaryArtifactData{}, fmt.Errorf("%w", err)
 	}
 
+	// Indices of any gradle-wrapper.jar files
+	var gradleWrappers []int
```

Comment:
I know I'm a java man, but would it make sense to call these `gradleWrapperIndexes` (or `gwIndexes` I dunno)?